sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -101,6 +109,8 @@ async fn run() -> Result<()> {
 
         // awsvpc mode is always available
         task_eni_enabled: true,
+
+        gpu_support_enabled: GPU_SUPPORT_ENABLED,
```

Comment:
nit: I wonder if we can make the condition more clear by using the [`cfg!`](https://doc.rust-lang.org/std/macro.cfg.html) macro here:
```suggestion
        gpu_support_enabled: !(cfg!(variant_type = "general_purpose")),
```

then we can ditch the intermediary const bool.

Reply:
I can give this a spin :+1: